app/build.gradle

```diff
@@ -23,7 +23,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
Should we really be mimifying for an application still in development?

Reply:
Yes I agree what does this have to with a currency converter ??

Reply:
I don't think it would benefit much while under development. This change seemed to resolve an issue preventing the CI from building.

Reply:
If that's the case then keep it as is.